templates/shared/forms/_form-validation.html

```diff
@@ -1,9 +1,9 @@
 <!-- form validator-->
 
 <!-- dependencies -->
-<script src="https://assets.ubuntu.com/v1/ec520d10-XMLHttpRequest.min.js"></script>
-<script src="https://assets.ubuntu.com/v1/4176b39e-serialize.js"></script>
-<script src="https://assets.ubuntu.com/v1/6b7597df-event-listener-polyfill.js"></script>
+<script src="https://assets.ubuntu.com/v1/ec520d10-XMLHttpRequest.min.js" defer></script>
+<script src="https://assets.ubuntu.com/v1/4176b39e-serialize.js" defer></script>
+<script src="https://assets.ubuntu.com/v1/6b7597df-event-listener-polyfill.js" defer></script>
```

Comment:
Could you caniuse these and see if we need these?

Reply:
https://caniuse.com/#search=XMLHttpRequest - can be removed

Reply:
https://github.com/nbouvrette/eventListenerPolyfill - can be removed